crates/forge/src/init.rs

```diff
@@ -0,0 +1,104 @@
+use anyhow::{bail, Context, Ok, Result};
+
+use include_dir::{include_dir, Dir};
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use toml_edit::{ArrayOfTables, Document, Item, Table};
+
+static TEMPLATE: Dir = include_dir!("starknet_forge_template");
+
+fn overwrite_files_from_template(
+    dir_to_overwrite: &str,
+    base_path: &PathBuf,
+    project_name: &str,
+) -> Result<()> {
+    let copy_form_dir = TEMPLATE.get_dir(dir_to_overwrite);
+    match copy_form_dir {
```

Comment:
Do we have a guarantee that Scarb basic template will have files only in `src` and `tests` folders so we replace them all with our files? cc @mkaput

Reply:
You can take this as guaranteed

Reply:
słowo harcerza